example/spec/parallel/discoverGranules/DiscoverGranulesHttpsSpec.js

```diff
@@ -25,37 +25,31 @@ const executionModel = new Execution();
 
 // Note: This test runs in serial due to the logs endpoint tests
 
-describe('The Discover Granules workflow with https Protocol', () => {
+// Disabled until we're acutally using https
```

Comment:
Do we not have https support?

Reply:
No, we don't have an HTTPS provider to download from in our integration tests.